python/paddle/fluid/executor.py

```diff
@@ -724,6 +725,9 @@ def run(self,
                 the input program is :code:`fluid.Program`, and the parameters(program, feed variable name
                 and fetch_list variable) of this interface remains unchanged during running.
                 The default is False.
+            return_merged(bool): This parameter indicates whether fetched variables (the variable specified
```

Comment:
Add some examples to show the usage of this parameter?

Reply:
Done.